[2022-02-09 09:01:20,494] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-08-01T23:00:00+00:00 [queued]>
[2022-02-09 09:01:20,512] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-08-01T23:00:00+00:00 [queued]>
[2022-02-09 09:01:20,513] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:01:20,513] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:01:20,513] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:01:20,540] {taskinstance.py:1259} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2021-08-01 23:00:00+00:00
[2022-02-09 09:01:20,550] {standard_task_runner.py:52} INFO - Started process 549 to run task
[2022-02-09 09:01:20,562] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'bigquery_external_table_task', 'scheduled__2021-08-01T23:00:00+00:00', '--job-id', '515', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmpuxpar8tm', '--error-file', '/tmp/tmpemxlke91']
[2022-02-09 09:01:20,564] {standard_task_runner.py:77} INFO - Job 515: Subtask bigquery_external_table_task
[2022-02-09 09:01:20,666] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-08-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 09:01:20,733] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:01:20,774] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-01T23:00:00+00:00
[2022-02-09 09:01:20,776] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-09 09:01:21,850] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=bigquery_external_table_task, execution_date=20210801T230000, start_date=20220209T090120, end_date=20220209T090121
[2022-02-09 09:01:21,901] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:01:21,940] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:17:08,503] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-08-01T23:00:00+00:00 [queued]>
[2022-02-09 09:17:08,519] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-08-01T23:00:00+00:00 [queued]>
[2022-02-09 09:17:08,520] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:17:08,520] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:17:08,520] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:17:08,535] {taskinstance.py:1259} INFO - Executing <Task(BigQueryCreateExternalTableOperator): bigquery_external_table_task> on 2021-08-01 23:00:00+00:00
[2022-02-09 09:17:08,543] {standard_task_runner.py:52} INFO - Started process 712 to run task
[2022-02-09 09:17:08,551] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'bigquery_external_table_task', 'scheduled__2021-08-01T23:00:00+00:00', '--job-id', '620', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmpdd3k58hv', '--error-file', '/tmp/tmp6bhnocn_']
[2022-02-09 09:17:08,553] {standard_task_runner.py:77} INFO - Job 620: Subtask bigquery_external_table_task
[2022-02-09 09:17:08,637] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.bigquery_external_table_task scheduled__2021-08-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:17:08,704] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:17:08,726] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=bigquery_external_table_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-01T23:00:00+00:00
[2022-02-09 09:17:08,728] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2022-02-09 09:17:10,222] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=bigquery_external_table_task, execution_date=20210801T230000, start_date=20220209T091708, end_date=20220209T091710
[2022-02-09 09:17:10,253] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:17:10,296] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
